extern/boostd-data/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/couchbase/gocb/v2 v2.5.0
-	github.com/docker/docker v20.10.7+incompatible
+	github.com/docker/docker v20.10.17+incompatible
```

Comment:
There are a bunch of changes to versions in go.mod and go.sum. Let's not change any versions unless we need to.

Reply:
I have changed the dependency back but require{} are automatically updated by go mod tidy as per requirements.